### ISSUE TITLE
Workaround to open OMEMO encrypted images/attachments?

### ISSUE THREAD

**Issue by boroli** [poVoq/conversejs-ubports#11]
Is there any workaround to open OMEMO encrypted images? I only get the a link like:

aesgcm://upload.quicksy.im/... 

When I click on it, nothing happens. I can't even copy it to the Morph browser or terminal to download it with wget. Is that related to the App or another security setting of UT itself?
This upstream bug seems to be related to it: https://github.com/conversejs/converse.js/issues/1182

I was just wondering how you can send files/images according to the screenshots in open store. Are the attachments then only working without OMEMO encryption?

**Comment by poVoq** [comment]
Sorry I think this is not supported by upstream ConverseJS for now. And yes the files send with http-upload as seen in the open-store screenshots are send unencrypted.

Copy&paste options are limited by the security and embedded browser design of UT I think.

**Comment by poVoq** [comment]
Small update: the next version of ConverseJS has the ability to receive but not send OMEMO encrypted attachments. I will update this app as soon as the new version (likely called 8.0) is released.

**Comment by lindsayb37** [comment]
@poVoq Any idea when 8.0 will be out? OMEMO encryption would definitely be a nice addition.

**Comment by poVoq** [comment]
OMEMO text encryption mostly works already in the current version. File attachments are the problem.

But I am not sure if I will update this app anytime soon. ConverseJS is super buggy in this use-case and I am rather hoping to use Movim soon when the OMEMO support for it reaches the stable version.

In addition my Nexus5 broke and my OnePlus3 is really not that well supported by UT...

Pull requests of course welcome.

**Comment by lindsayb37** [comment]
How do I use OMEMO for simple messages. I tried but it just pops up a message saying I received an encrypted message but my client doesn't support it.

**Comment by poVoq** [comment]
hmm, odd. Normally it should start exchanging OMEMO keys automatically at least after sending a message. Did you try adding the other side to your roster? I can't really help you debugging it, but 1:1 chat OMEMO generally works in ConverseJS.

**Comment by lindsayb37** [comment]
What is a roster? Is that just contacts?

**Comment by poVoq** [comment]
Yes those contacts you have in your list and have approved presence exchange etc.
